src/contracts/GPv2SimpleAuthentication.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "./interfaces/GPv2Authentication.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2SimpleAuthentication is Ownable, GPv2Authentication {
```

Comment:
nit: I personally am not a fan of the word simple for components (as it's not at all descriptive). How about GPv2AllowListAuthentication ?

Reply:
I suppose you would say the same about "Basic" or "Elementary". I am also fine with "AllowList"

Reply:
Went with AllowList